protocols/mdns/src/behaviour.rs

```diff
@@ -115,31 +119,59 @@ pub struct Mdns {
 
     /// Discovery timer.
     timeout: Timer,
+
+    // Mutlicast address.
+    multicast_addr: IpAddr,
+
+    // Socket address.
+    socket_addr: SocketAddr,
```

Comment:
do we need the socket_addr for anything?

Reply:
it is used in line 397, might be created there but wanted avoid not needed object creation